src/templates/repository/index.jsx

```diff
@@ -83,8 +89,39 @@ const RepositoryPage = ({ data, location }) => {
       <article className="repository">
         <header className="repository__hero">
           <Nav />
-          <RepositoryTitle ownerName={ownerName} name={name} tag="h1" />
-          <p>{description}</p>
+
+          <div className="repository__header">
+            <RepositoryTitle ownerName={ownerName} name={name} tag="h1" />
+            <div className="repository__meta">
```

Comment:
Looks perfect @delbetu !

There is one thing I would do since the code is super similar to the one of the card.

We could include all the elements to the `RepositoryTitle` component and rename it to `RepositoryMeta` or something.

The component could exactly this content which is the title, description, stars and dates. What do you think?

Then the component could be called here and on the `Card` component in the same way.

Reply:
sounds good :+1: